.github/bin/run-clang-tidy.sh

```diff
@@ -15,6 +15,11 @@
 
 set -u
 
+if [ ! -e WORKSPACE ]; then
```

Comment:
or `[ ! -f WORKSPACE ]` (test if exists and is file)

Reply:
Thanks. Done.